packages/firestore/src/util/api.ts

```diff
@@ -41,11 +44,15 @@ export function makeConstructorPrivate<T extends Function>(
     throw new FirestoreError(Code.INVALID_ARGUMENT, error);
   }
 
-  // Make sure instanceof checks work and all methods are exposed on the public
-  // constructor
-  PublicConstructor.prototype = cls.prototype;
+  // Copy static members and prototype
+  for (const staticProp of Object.getOwnPropertyNames(cls)) {
+    if (RESERVED_READONLY_PROPS.indexOf(staticProp) === -1) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (PublicConstructor as any)[staticProp] = (cls as any)[staticProp];
+    }
+  }
 
-  // Copy any static methods/members
+  // Copy instance methods/members
   Object.assign(PublicConstructor, cls);
```

Comment:
We don't need it anymore since instance methods/members are copied in `prototype`.